gobdd_test.go

```diff
@@ -83,6 +85,32 @@ func TestInvalidFunctionSignature(t *testing.T) {
 	}
 }
 
+func TestFailureOutput(t *testing.T) {
+	testCases := []struct {
+		name     string
+		f        interface{}
+		expected []string
```

Comment:
expected what? I'd put a more clear name of it. Expected errors? `expectedErrs`?

Reply:
Good suggestion!